### ISSUE TITLE
Logitech/Elgato integration

### ISSUE THREAD

**Issue by Spenser715** [nvdweem/PCPanel#30]
I'm not sure what all APIs are available but it would be great to be able to use a knob or slider to control lights such as the Logitech litra glow. I'm sure there are other lights that would work well with this.

**Comment by nvdweem** [comment]
Lights and such tend to have good api's, but I don't have any of those which will make building and testing it hard.

**Comment by Spenser715** [comment]
I don't mind testing for the Logitech litra lights. They are the only ones I have to be able to test.

**Comment by nvdweem** [comment]
I've looked at Elgato Wave Link, from what I can tell, the API is not documented. There does seem to be a .net project where they are reverse engineering the API, but that seems to break every now and then.

It seems that the route to proxy the api between the Stream Deck and Wave Link still works. 
`tproxy -r 127.0.0.1:1885 -p 1884` seems to show all the messages being sent and received. So reverse engineering should still be possible.

I'll try and see if I have any time to play around with this.

**Comment by nvdweem** [comment]
> but I know it is probably a nightmare getting this to work with Wave Audio software.

Starting to see that now, they _used_ to open port 1884 on the WaveLink software. Nice, easy, stable.
Now they have a json file @ `process.env.APPDATA?.replace("Roaming", "Local/Packages/Elgato.WaveLink_g54w8ztgkx496/LocalState/ws-info.json")` which holds the port that wavelink uses 🥇

Anyway, WaveLink integration should be working again. I noticed that wavelink is now populating my non-elgaty microphones, so I should be able to add (some) support for the input channels as well.

Edit: Apparently it's not stable between reboots, and the PCPanel software might boot up too early. If that happens, restarting the software is needed (for now). Will change it so that it will re-read the port when reconnecting.

**Comment by FaNt4zMa** [comment]
> I noticed that wavelink is now populating my non-elgaty microphones
Yeah thats what I meant when I said they released it publicly... 😅 

Once WaveLink support is stable, I'm looking forward to switch from Voicemeeter 😂 re-configuring is gonna be a pain but I'm hoping it will be more stable. I have this issue with Voicemeeter where when I launch a game (or something quite CPU intensive happens), it "crashes" the audio engine and I have to restart the software completely.

Anyway thanks for the amazing work you're doing 🙏
